cardigan_beta/.storybook/config.js

```diff
@@ -7,10 +7,12 @@ function loadStories() {
   const pages = require.context('../stories/pages', true, /\.js$/);
   const components = require.context('../stories/components', true, /\.js$/);
   const global = require.context('../stories/global', true, /\.js$/);
+  const wip = require.context('../stories/wip', true, /\.js$/);
 
   pages.keys().forEach((filename) => pages(filename));
   components.keys().forEach((filename) => components(filename));
   global.keys().forEach((filename) => global(filename));
+  wip.keys().forEach((filename) => wip(filename));
```

Comment:
To be a stickler - is it worth having something more precise, as in they're aren't WIP, they're just ported over.

Something like `The wild`,  or `unclean`, `unhealthy`?

I ask as WIP feels like it would be useful for exactly that type of work.
Then they can exist in the components directory, just have a WIP prefix.